src/components/home/home.js

```diff
@@ -1,17 +1,37 @@
-import { HomeGraphics, HomePic, HomePicContainer, HomePicText, HomePicTextContainer, HomeSection, Title, TitleSpan } from "./styledHome";
+import { Handle, HomeGraphics, HomePic, HomePicContainer, HomePicText, HomePicTextContainer, HomeSection, Title, TitleSpan } from "./styledHome";
 import trash from '../../images/TrashPlaceHolder.png';
+import container0 from '../../images/container-0-copy.png';
+import handle from '../../images/sliderHandle.svg';
+import ReactCompareImage from 'react-compare-image';
 import Animation from "../animation/Animation";
+import React from "react";
 
 const Home = () => {
 
     return (
         <HomeSection>
-            <Title>Help our environment by eliminating <br /> single-use plastics from your delivery with<br />
-                <TitleSpan>Awesome Container Company</TitleSpan>
+            <Title>Help our environment by eliminating single-use plastics from your delivery with
+                <TitleSpan> Awesome Container Company</TitleSpan>
             </Title>
             <HomeGraphics>
                 <HomePicContainer>
-                    <HomePic src={trash} alt='trash' />
+                    <HomePic>
+                        <ReactCompareImage
+                            handle={<Handle src={handle} />}
+                            leftImage={trash}
+                            leftImageCss={
+                                { height: '100%' }
+                            }
+                            rightImage={container0}
+                            rightImageCss={
+                                {
+                                    height: '100%'
+                                }
+                            }
+                            sliderLineColor='#FF9820'
+                            handleSize='28'
+                        />
+                    </HomePic>
```

Comment:
I think the text 'now' and 'with awesome containers' should be inside the homePicContainer. Look at 1440px design and notice the text lines up with the bottom of the animation. Right now the text is under it.